### PR TITLE
community/lld: upgrade to 7.0.1

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by ncopa** [comment]
this was fixed with a84ab695096337553ae79182143730853331e0e5

**Comment by ncopa** [comment]
I think we should rebase this and update commit message. The use of extranal lit looks better than we did in a84ab69.

File: community/lld/APKBUILD [hunk 1]
```diff
@@ -33,6 +33,7 @@ build() {
 		-DCMAKE_INSTALL_PREFIX=/usr \
 		\
 		-DCMAKE_SKIP_INSTALL_RPATH=ON \
+		-DLLVM_EXTERNAL_LIT=/usr/lib/llvm7/bin/lit \
```
Comment (ncopa) [inline community/lld/APKBUILD]: Can we also use somthing like  `-DLLVM_EXTERNAL_LIT=/usr/lib/llvm${pkgver%%.*}/bin/lit` here?

Comment (J0WI) [reply community/lld/APKBUILD]: Tests are not passing as expected. Any hints?

**Comment by J0WI** [comment]
a84ab695096337553ae79182143730853331e0e5

**Pull request closed without merging**